core/ctors.go

```diff
@@ -0,0 +1,17 @@
+package core
+
+type (
+	// SourceCtor is a map of string to source constructors.
```

Comment:
I agree, the word "map" is problematic here. I also think that using `ctor` is also problematic because Go doesn't have constructors, so the naming and comments of this whole section feels off to me.

I will create an issue to rename/cleanup this section.

Reply:
https://github.com/rename-this/vhs/issues/37